pavelib/prereqs.py

```diff
@@ -22,6 +22,7 @@
 PYTHON_REQ_FILES = [
     'requirements/edx/pre.txt',
     'requirements/edx/github.txt',
+    'requirements/edx/stanford.txt',
```

Comment:
I think this is the only change we need here. This should add it to the watch/fingerprint list.

Granted, I think it will try to install `stanford.txt` twice (once individually, another as part of `github.txt`, but the 2nd run should end up being a NOOP.